example/playlist.html

```diff
@@ -0,0 +1,439 @@
+        playbackState: null,
+      };
+
+      const isPlayerState = (state) => () => STATE.playbackState === state;
+
+      const isPlaying = isPlayerState('PLAYING');
+      const isPaused = isPlayerState('PAUSED');
+
+      // Whenever the playhead updates, we want to update the length of the
+      // progress bar. This callback is not called aggressively, so the
+      // resulting progress bar will have a "ticking" behavior versus smooth
+      // updates.
+      const onPlaybackProgress = (progress) => {
+        let currentTrack = TRACKS[STATE.currentTrackID];
+        let currentTrackBytes = currentTrack.fileSize * progress;
+        let previousTrackBytes = TRACKS.slice(0, STATE.currentTrackID).reduce(
+          (xs, x) => xs + x.fileSize,
+          0
+        );
+
+        STATE.progress =
+          ((currentTrackBytes + previousTrackBytes) / TOTAL_MIX_BYTES) * 100;
+
+        render();
+      };
+
+      const onPlaybackEnded = () => {
+        player.reset();
+        STATE.currentTrackID = null;
+        STATE.playbackState = null;
+        STATE.progress = 0.0;
+
+        render();
+      };
+
+      // Whenever a new track begins playing, we update the UI state and re-render.
+      const onTrackChanged = (currentTrack, nextTrack) => {
+        STATE.currentTrackID = nextTrack.meta.id;
+        render();
+      };
+
+      // Here we actually initialize the Player. It doesn't require any
+      // arguments, but most people will want to pass in at least a couple of
+      // handlers to interact with its errors and ready state.
+      const player = new ProtonPlayer({
+        onReady: () => {
+          STATE.ready = true;
+          render();
+        },
+        onError: (e) => console.error(e),
+        onPlaybackEnded,
+        onPlaybackProgress,
+        onTrackChanged,
+      });
+
+      const startPlayback = () => {
+        const firstTrack = TRACKS[0];
+        STATE.currentTrackID = firstTrack.meta.id;
+        STATE.playbackState = 'PLAYING';
+
+        player.play(TRACKS);
+        render();
+      };
+
+      const updateElement = (className, f) => {
+        const element = document.body.getElementsByClassName(className)[0];
+        f(element);
+      };
+
+      const updateElements = (className, f) => {
+        const elements = document.body.getElementsByClassName(className);
+        elements.forEach(f);
+      };
+
+      const render = () => {
+        updateElement('ProgressBar', (e) => {
+          e.style.width = `calc(${STATE.progress}% - 4px)`;
+        });
+
+        updateElement('NowPlaying_Image', (e) => {
+          if (STATE.currentTrackID != null) {
+            e.style.backgroundImage = `url(${
+              TRACKS[STATE.currentTrackID].meta.coverArt
+            })`;
+          } else {
+            e.style.backgroundImage = 'url(./ProtonPlayer.gif)';
+          }
+        });
+
+        updateElement('Controls_PlayPause', (e) => {
+          e.textContent = isPlaying() ? '⏸' : '⏵';
```

Comment:
The second emoji is not visible for me on Linux. Maybe we need to consider changing it? We can see once this is deployed on GitHub Pages.